packages/contracts-bedrock/src/EAS/EAS.sol

```diff
@@ -40,7 +40,7 @@ struct AttestationsResult {
     bytes32[] uids; // UIDs of the new attestations.
 }
 
-/// @custom:proxied
+/// @custom:proxied true
```

Comment:
Had to be updated to @Custom:proxied true because the Solidity compiler generates natspec assuming there's a value field, which resulted in an output like` "proxied": "@custom:predeploy 0x4200000000000000000000000000000000000021"` (eating the line below it).